### PR TITLE
Correct and Enhance `#qr_test` (Closes #898 & #899)

### LINKED ISSUE #898: Wrong wording in `#qr_test`
## Where to find the issue
www.coronawarn.app/en/faq/#qr_test:

https://github.com/corona-warn-app/cwa-website/blob/7f7cd0c1bbd7df8d562135c442bbade05f11a72d/src/data/faq.json#L159

says: 
`[...]to warn your risk encounters.[...]`

www.coronawarn.app/de/faq/#qr_test:

https://github.com/corona-warn-app/cwa-website/blob/7f7cd0c1bbd7df8d562135c442bbade05f11a72d/src/data/faq_de.json#L159

says:
`[...]um über die Corona-Warn-App Ihre Risikobegegnungen zu warnen.[...]`

## Describe the issue

1. The texts don't match 
2. You can't warn your risk encounters/Risikobegegnungen, you warn your contacts
                           
## Suggested change

EN: 
 
"to warn your risk encounters" 
to 
"to warn your contacts via the Corona-Warn-App"

DE:

"um über die Corona-Warn-App Ihre Risikobegegnungen zu warnen"
to
"um über die Corona-Warn-App Ihre Kontakte zu warnen"

## Additional information 

PR #900

### LINKED ISSUE #899: Hard to understand wording in `#qr_test`
## Where to find the issue
https://www.coronawarn.app/de/faq/#qr_test
https://www.coronawarn.app/en/faq/#qr_test

## Describe the issue
The FAQ Entry says that you should report an infection via the hotline if you are tested positive, then it gives advice how to do this.

## Suggested change
We should also explain to the user how to remove the still pending test result, since the user can only enter a TAN after deleting the pending result

## Additional information
PR #900

### PR DESCRIPTION
**Pull request by Ein-Tim**
This PR corrects wrong wording in `#qr_test` (#898) and enhances the entry to explain how to delete the test result before entering the TeleTAN (#899).

<details>
<summary>Screenshots of the changes</summary>

DE:
image

EN:
image

</details>

---

Closes #898 
Closes #899

### CODE REVIEW

File: src/data/faq_de.json [hunk 1]
```diff
@@ -156,9 +156,11 @@
...
-                            "Rufen Sie bitte beim zuständigen ärztlichen Fachpersonal bzw. Testcenter/Gesundheitsamt an, um Ihr Testergebnis zu erhalten. Wird Ihnen dabei ein positives Testergebnis mitgeteilt, können Sie über die Hotline unter der 0800 7540002 eine TeleTAN beantragen, um über die Corona-Warn-App Ihre Risikobegegnungen zu warnen. Tragen Sie für diesen Zweck die erhaltene TeleTAN unter",
+                            "Rufen Sie bitte beim zuständigen ärztlichen Fachpersonal bzw. Testcenter/Gesundheitsamt an, um Ihr Testergebnis zu erhalten. Wird Ihnen dabei ein positives Testergebnis mitgeteilt, können Sie über die Hotline unter der 0800 7540002 eine TeleTAN beantragen, um über die Corona-Warn-App Ihre Kontakte zu warnen. Bevor Sie die TeleTAN in die App eintragen können, müssen Sie erst das noch ausstehende Testergebnis aus der App löschen. Dazu tippen Sie bitte auf ",
+                            "<ul><li>'Test anzeigen' ->  'Test entfernen'</li></ul>",
+                            "Bestätigen Sie danach noch einmal das Sie den Test wirklich entfernen wollen. Nun können Sie die erfragte TeleTAN unter",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:161]: einmal das Sie => einmal, dass Sie

Comment (Ein-Tim) [reply src/data/faq_de.json]: Corrected

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
In Android the labels:
DISPLAY TEST
DELETE TEST
NEXT STEPS
are displayed in upper case, so if this is different on iOS then choose either all upper case or all mixed cased for these labels (and their equivalent German names).

**Comment by Ein-Tim** [comment]
@MikeMcC399 

Yes this differs from iOS, I will make all labels mixed cased.

File: src/data/faq_de.json [hunk 2]
```diff
@@ -156,9 +156,11 @@
...
-                            "Rufen Sie bitte beim zuständigen ärztlichen Fachpersonal bzw. Testcenter/Gesundheitsamt an, um Ihr Testergebnis zu erhalten. Wird Ihnen dabei ein positives Testergebnis mitgeteilt, können Sie über die Hotline unter der 0800 7540002 eine TeleTAN beantragen, um über die Corona-Warn-App Ihre Risikobegegnungen zu warnen. Tragen Sie für diesen Zweck die erhaltene TeleTAN unter",
+                            "Rufen Sie bitte beim zuständigen ärztlichen Fachpersonal bzw. Testcenter/Gesundheitsamt an, um Ihr Testergebnis zu erhalten. Wird Ihnen dabei ein positives Testergebnis mitgeteilt, können Sie über die Hotline unter der 0800 7540002 eine TeleTAN beantragen, um über die Corona-Warn-App Ihre Kontakte zu warnen. Bevor Sie die TeleTAN in die App eintragen können, müssen Sie erst das noch ausstehende Testergebnis aus der App löschen. Dazu tippen Sie bitte auf ",
+                            "<ul><li>'Test anzeigen' ->  'Test entfernen'</li></ul>",
+                            "Bestätigen Sie danach noch einmal, das Sie den Test wirklich entfernen wollen. Nun können Sie die erfragte TeleTAN unter",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:161]: einmal, das
should be
einmal, dass

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
This is about "einmal, dass"

https://www.duden.de/sprachwissen/sprachratgeber/das-oder-dass

File: src/data/faq_de.json (hunk 2 above)
Comment (Ein-Tim) [reply src/data/faq_de.json]: Corrected

**Review by heinezen — APPROVED** [review]
LGTM

**Pull request merged**